src/django/api/models.py

```diff
@@ -42,16 +42,16 @@ def create_superuser(self, email, password, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
 
-class Organization(models.Model):
+class Contributor(models.Model):
     """
     A participant in or observer of the supply chain that will
     upload facility lists to the registry.
     """
     # These choices must be kept in sync with the identical list kept in the
     # React client's constants file
-    OTHER_ORG_TYPE = 'Other'
+    OTHER_CONTRIB_TYPE = 'Other'
```

Comment:
Originally I had thought we might have to make a corresponding change in the React client's constants file, but it turns out the only thing sent back here is `('Other', 'Other')`. I wonder if it might make sense just to call this `OTHER_TYPE` or something even more vague.

Reply:
I think this does some minor typing for us, so we don't mix-up `OTHER_CONTRIB_TYPE` with `OTHER_SOMETHINGELSE_TYPE`, even though they may both be `'Other'` as strings.